### DISCUSSION TITLE
Solar Export Monitoring

### DISCUSSION THREAD

**Discussion by pendlew** [HandyHat/ha-hildebrandglow-dcc#225] [Ideas]
Would it be possible to add a sensor to log the Solar Export energy?
My meter shows an Export counter.

**Comment by HandyHat** [comment]
Unfortunately, unless you are using Hildebrand hardware (which provides the export readings via MQTT), the Hildebrand API does not currently support export readings - as described [on their forums](https://forum.glowmarkt.com/index.php?p=/discussion/comment/595#Comment_595), this is being worked on by them.

**Comment by donaldwpage** [comment]
I'm using the Hildebrand hardware and this integration. I keep getting negative consumption so would really like this to work. Happy to test and provide logs

**Comment by Sodgester** [comment]
Hi, looks like the export API is now live: recent comment from Hildebrand "Support for export is now underway. We've done the work to pick up the export (and it is great with SMETS2 meters as export is on by default) and store the data and then play it back into our APIs (PWR_EXP). This work is all done by us as a side activity so it has to fit in with commercial priorities. If you take the MQTT feed apparently the export stream is in there. Over the next months we will be adding Export to Bright and glowmarkt."
Any plans to include this into the integration? Thanks!

**Reply by ItsIllak** [reply]
Wondering the same - in the Hildebrand, I can see "active export", but get no devices or entities in Home assistant.

**Reply by Sodgester** [reply]
I ended up buying their CAD. That works great with Home Assistant.
